test/v1/test_account_info.py

```diff
@@ -292,11 +292,39 @@ def test_upgrade_2_default_app_key(self):
     def _make_info(self):
         return self._make_sqlite_account_info()
 
-    def _make_sqlite_account_info(self, last_upgrade_to_run=None):
+    def _make_sqlite_account_info(self, env=None, last_upgrade_to_run=None):
         """
-        Returns a new StoredAccountInfo that has just read the data from the file.
+        Returns a new SqliteAccountInfo that has just read the data from the file.
         """
-        return SqliteAccountInfo(file_name=self.db_path, last_upgrade_to_run=None)
+        return SqliteAccountInfo(
+            file_name=self.db_path if not env else None,
+            env=env or {},
+            last_upgrade_to_run=last_upgrade_to_run
```

Comment:
please add a trailing `,`  in this line so that `yapf` reliably guards the format of the multi-line statement